packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -200,13 +200,19 @@ class SizeTransition extends AnimatedWidget {
   /// width or height multiplied by v.
   final Animation<double> sizeFactor;
 
-  /// How to align the child. See the [Align] widget.
-  final FractionalOffset alignment;
+  /// How to align the child along the axis that sizeFactor is modifying.
+  /// See the [Align] widget.
```

Comment:
We probably shouldn't refer to [Align] here because the parameter isn't the same.  We can probably just remove this sentence.